### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by Daanikus**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,143 +1,83 @@
...
+## GitHub
 
-+ `@CodeLingoBot review` reviews the PR based on default Rules for the repo's language or custom Rules added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@review`</summary>
-    + `@CodeLingoBot review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
-    + `@CodeLingoBot review un-ignore` un-ignores the issue for current and future reviews.
-+ `@CodeLingoBot rewrite` makes a PR to this PR fixing the issues found based on default Rules for the repo's language or custom Rules added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@rewrite`</summary>
-  
-    + `@CodeLingoBot rewrite set <varname>` sets the value of a variable in a rewrite template.
+To have CodeLingo automatically review every new Pull Request to your repo, [install CodeLingo on your repo](https://github.com/marketplace/codelingo) and add some rules to your repo via the [dashboard](https://dash.codelingo.io).
```
Comment (fwereade) [inline README.md:37]: is this where we want to focus? isn't it about living docs – capture rules and discuss them is what we can _actually_ do right now

**Review by fwereade — APPROVED** [review]
It's unquestionably _better_ so LGTM on that basis but I'd appreciate @waigani 's input as well.

File: README.md (hunk 1 above)
Comment (waigani) [reply README.md:37]: +1

File: README.md [hunk 2]
```diff
@@ -1,143 +1,83 @@
...
+</h3>
 <p align="center">
-  <b>Drive Continuous Higher Standards with CodeLingo's Automated Reviews, Bug Fixes and Docs! </b>
+  Stop making linters happy. Start making your team productive.
 </p>
 
 <p align="center">
-  <a href="https://github.com/apps/codelingo" target="_blank">
-    <img width="295" height="38" src="https://raw.githubusercontent.com/codelingo/codelingo/master/public/img/install.png" />
+  <a href="https://github.com/marketplace/codelingo" target="_blank">
```
Comment (waigani) [inline README.md:13]: Keep apps URL as per @rstorr's comment

File: README.md [hunk 3]
```diff
@@ -1,143 +1,83 @@
-
-
 <h3 align="center"> <a href="https://codelingo.io" target="_blank">
-    <img src="./public/img/CLLogo.svg" />
+    <img width="500" alt="logo" src="./public/img/codelingologobig.svg" />
```
Comment (waigani) [inline README.md:2]: Checking this is the new logo?

**Review by waigani — APPROVED** [review]
- remove "at at" from VS code screenshot
- screenshot of PR capture comment (with bot reply)
- screenshot of replying to a bot comment and that comment appearing in the rule thread (side by side, the rule thread should show some team discussion on it too)

**Pull request merged**